development/README.md

```diff
@@ -33,3 +33,6 @@ If you change the value of the `DATABASE` variable at any time during developmen
     - `docker volume rm <volume-id>`
 3. rebuild the docker image
     - `docker-compose up --build -d`
+
+## Adding hostfile
+ To make use of the container mailhog you need to create a hosts file entry you can do this by editing `/etc/host` make sure that this record matches the variable in `.env`called HOSTNAME
```

Comment:
This is an example of accidental complexity added by this pull request. 

We are adding a step to developing this theme by requiring the user to make system-wide configuration changes. We are also adding a documentation requirement (presumably for *nix and Windows developers to edit their `hosts` file.)

While the overall complexity is relatively low, it is not addressing any direct need. Further, this type of complexity tends to grow by accretion.

Reply:
> This is an example of accidental complexity added by this pull request.
> 
> We are adding a step to developing this theme by requiring the user to make system-wide configuration changes. We are also adding a documentation requirement (presumably for *nix and Windows developers to edit their `hosts` file.)
> 
> While the overall complexity is relatively low, it is not addressing any direct need. Further, this type of complexity tends to grow by accretion.

in this example i have done this because you cant set your email @localhost in wordpress this shall return in an error, so i had to add some virtual host in the host file. To resolve this issue this step is not a requirement only a recomendation.